apps/opencs/model/prefs/shortcuteventhandler.cpp

```diff
@@ -49,7 +49,7 @@ namespace CSMPrefs
         ShortcutMap::iterator shortcutListIt = mWidgetShortcuts.find(widget);
         if (shortcutListIt != mWidgetShortcuts.end())
         {
-            std::remove(shortcutListIt->second.begin(), shortcutListIt->second.end(), shortcut);
+            shortcutListIt->second.erase(std::remove(shortcutListIt->second.begin(), shortcutListIt->second.end(), shortcut), shortcutListIt->second.end());
```

Comment:
This looks like another Qt-ownership issue.

Reply:
`std::remove` doesn't actually remove anything, it just moves the matching entries to the end of the container and returns an iterator to the first of the entries to remove. It would seem quite odd if this intended to only move the given widget's shortcuts to the end of the container but leave them in it. In either case, this isn't deleting anything, just removing the pointer(s) from the widget's map entry.

Reply:
Hm, right. I misread that.